### DISCUSSION TITLE
streaming-v4-unsigned.go

### DISCUSSION THREAD

**Discussion by T-TRz879** [minio/minio#18879] [Q&A] [answered]
May I ask if there are any relevant documents in streaming-v4-unsigned.go

**Comment by klauspost** [comment, marked as answer]
It just follows the AWS S3 spec: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html

**Reply by T-TRz879** [reply]
Thanks bro!!
